client/my-sites/checkout/checkout-thank-you/use-get-jetpack-activation-confirmation-info.ts

```diff
@@ -0,0 +1,134 @@
+	image: string;
+	text: TranslateResult | ReactNode;
+	buttonUrl: string;
+};
+
+const useGetJetpackActivationConfirmationInfo = (
+	siteId: number,
+	productSlug: string
+): ActivationConfirmationInfo => {
+	const translate = useTranslate();
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) );
+	const wpAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId ) );
+
+	const baseJetpackCloudUrl = 'https://cloud.jetpack.com';
+
+	const confirmationInfo = {
+		jetpack_anti_spam: {
+			image: successImageAntiSpam,
+			text: translate(
+				"We'll take care of everything from here. Now you can enjoy a spam-free site!"
+			),
+			buttonUrl: wpAdminUrl || baseJetpackCloudUrl,
+		},
+		jetpack_backup: {
+			image: successImageDefault,
+			text: translate(
+				'You can see your backups and restore your site on {{a}}cloud.jetpack.com{{/a}}. If you ever lose access to your site, you can restore it there.',
+				{
+					components: {
+						a: createElement( 'a', { href: baseJetpackCloudUrl } ),
+					},
+				}
+			),
+			buttonUrl: siteSlug
+				? `${ baseJetpackCloudUrl }/backup/${ siteSlug }`
+				: `${ baseJetpackCloudUrl }/landing`,
+		},
+		jetpack_complete: {
+			image: successImageComplete,
+			text: translate(
+				'You can see your backups, security scans, and restore your site on {{a}}cloud.jetpack.com{{/a}}. If you ever lose access to your site, you can restore it there.',
+				{
+					components: {
+						a: createElement( 'a', { href: baseJetpackCloudUrl } ),
+					},
+				}
+			),
+			buttonUrl: siteSlug
+				? `${ baseJetpackCloudUrl }/backup/${ siteSlug }`
+				: `${ baseJetpackCloudUrl }/landing`,
+		},
+		jetpack_scan: {
+			image: successImageScan,
+			text: translate( 'You can see your security scans on {{a}}cloud.jetpack.com{{/a}}.', {
+				components: {
+					a: createElement( 'a', { href: baseJetpackCloudUrl } ),
+				},
+			} ),
+			buttonUrl: siteSlug
+				? `${ baseJetpackCloudUrl }/scan/${ siteSlug }`
+				: `${ baseJetpackCloudUrl }/landing`,
+		},
+		jetpack_search: {
+			image: successImageSearch,
+			text: translate( "Next, we'll help you customize your Search experience for your visitors." ),
+			buttonUrl: siteSlug
+				? `${ baseJetpackCloudUrl }/jetpack-search/${ siteSlug }`
+				: `${ baseJetpackCloudUrl }/landing`,
+		},
+		jetpack_security: {
+			image: successImageDefault,
+			text: translate(
+				'You can see your backups, security scans, and restore your site on {{a}}cloud.jetpack.com{{/a}}. If you ever lose access to your site, you can restore it there.',
+				{
+					components: {
+						a: createElement( 'a', { href: baseJetpackCloudUrl } ),
+					},
+				}
+			),
+			buttonUrl: siteSlug
+				? `${ baseJetpackCloudUrl }/backup/${ siteSlug }`
+				: `${ baseJetpackCloudUrl }/landing`,
+		},
+		jetpack_videopress: {
+			image: successImageDefault,
+			text: translate( 'Experience high-quality, ad-free video built specifically for WordPress.' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'High-quality, ad-free video built specifically for WordPress.' )` *ES Score: 9*